tools/mo/setup.py

```diff
@@ -102,6 +102,7 @@ def find_package_modules(self, package, package_dir):
       'openvino.tools.mo.front.mxnet': ['*.json'],
       'openvino.tools.mo.front.onnx': ['*.json'],
       'openvino.tools.mo.front.tf': ['*.json'],
+      'openvino.tools.mo.front.caffe': ['CustomLayersMapping.xml*']
```

Comment:
Why this change?

Reply:
This file is needed by Caffe models and by mistake was missed in when mo is installed through pip. It should be copied to wheel package.
And because this file skipping during check we didn't get any bom error even when they are turned on.

Reply:
This file is mentioned nowhere now. How is it used and for what are not clear? I did not find it in the documentation. Please add this in the documentation where we have an example with custom mapping. Let users see the example of such mapping.

Reply:
There are some documentation about CustomLayersMapping for Caffe. Exactly this file `CustomLayersMapping.xml.example` is not mentioned 
but `CustomLayersMapping.xml` is mentioned here https://github.com/openvinotoolkit/openvino/blob/master/docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Caffe.md

I didn't add this new file I just returned it back when it was at the package, e.g. at 2021.4 it was copied to model_optimizer during installation 
![image](https://user-images.githubusercontent.com/5703039/201616582-ec64093f-b89a-4eaf-9a07-7ccea49e522d.png)
but when building package switched to pip/wheels we lost it. Here I returned it back.

Reply:
Now this is useless file in our package. Let us make some value for this file by adding just one sentence about it. Believe - not a big effort:)

Reply:
I agree it's not a big effort :) 
I am afraid of the CI validation problems. I will add such line into documentation and try to push, maybe CI will see that only documentation was changed and will not full e2e validation. Let's see if this works :)

Reply:
CI is not a problem. During the weekend I obtained three PRs with green CI.

Reply:
Mentioned purpose of `CustomLayersMapping.xml.example` in the docs